device/mellanox/x86_64-nvidia_sn4280-r0/installer.conf

```diff
@@ -1 +1 @@
-ONIE_PLATFORM_EXTRA_CMDLINE_LINUX="libata.force=noncq module_blacklist=mlx5_ib,mlx5_core"
+ONIE_PLATFORM_EXTRA_CMDLINE_LINUX="libata.force=noncq module_blacklist=mlx5_ib,mlx5_core modprobe.blacklist=tpm_tis,tpm_crb,tpm"
```

Comment:
What's the difference between `module_blacklist` and `modprobe.blacklist` here?

Reply:
The `modprobe.blacklist` is used to be consistent with the fixes for the other platforms:
https://github.com/sonic-net/sonic-buildimage/pull/19221 [Mellanox] Disable TPM on SN5600 and SN5400 platform